keel-sql/keel-sql.gradle.kts

```diff
@@ -65,16 +67,37 @@ dependencies {
   liquibaseRuntime("mysql:mysql-connector-java")
 }
 
-tasks.register<Exec>("runMysql") {
-  commandLine("docker run --name mysqlJooq -d --rm -e MYSQL_ROOT_PASSWORD=sa -e MYSQL_DATABASE=keel -p 6603:3306 mysql:5.7".split(" "))
-  doLast {
-    // Wait for the DB server to come up...
-    sleep(15 * 1000)
-  }
+tasks.register<Exec>("startMysql") {
+  commandLine("sh", "-c",
+    "docker stop mysqlJooq || " +
```

Comment:
Hacky, but does the trick -- always try to stop the container just in case before starting.